iosApp/iosApp/Pages/TripDetails/TripDetailsPage.swift

```diff
@@ -71,9 +72,10 @@ struct TripDetailsPage: View {
     var body: some View {
         VStack(spacing: 16) {
             header
-            if let trip, let globalResponse, let vehicle = vehicleResponse?.vehicle,
+            if tripPredictionsLoaded, let globalResponse, let vehicle = vehicleResponse?.vehicle,
                let stops = TripDetailsStopList.companion.fromPieces(
```

Comment:
There are tests that assert that `fromPieces` will return something reasonable if the predictions haven't loaded but the schedules have. Are we certain we want to hide schedules if predictions have not loaded?

Reply:
Were you waiting on a reply to this? I tried to address it in the other thread.

Reply:
Oh. I might have been, or I might have just lost track of this PR.